account-kit/react/src/components/auth/card/main.tsx

```diff
@@ -14,7 +14,9 @@ export const MainAuthContent = () => {
   return (
     <>
       {header}
-      {showSignInText && <h3 className="font-semibold text-lg">Sign in</h3>}
+      {(showSignInText == null || showSignInText) && (
```

Comment:
lol this is slightly conufsing. So if sign in text is true or null we'd want to show signin text? Is there a better default we could use here?